### ISSUE TITLE
Enhancement: templates and better filter support for #raw

### ISSUE THREAD

**Issue by chatziko** [AlexxIT/go2rtc#487]
`#raw` is super useful, but currently has 2 limitations
- no template support
- cannot use `-vf` in cases when go2rtc adds its own filters (ffmpeg only accepts a single `-vf`)

So I propose the following:
1. Add template support: `#raw=my_template` should be expanded if a template `my_template` exists.
2. If raw (after expansion) contains `-vf` then go2rtc should __not__ insert its own filters, it's the user's responsibility to add all filters in the proper order.

A possible use case is #402, but I can imagine many others, raw/filters are very general features.

**Comment by AlexxIT** [comment]
`#filter=value` also good idea

**Comment by chatziko** [comment]
> `#filter=value` also good idea

Indeed, in fact I started writing this issue after my comment in #402 to suggest `#filter=value`.

But then I realized a problem: in what order should the user filters be combined with the generated ones? Say `#hardware` adds `hwupload`, I might want to add a filter either before or after. So maybe its easier to let the user supply all filters without merging, in which case `#raw` is already sufficient, no need to introduce `#filter`.
